### PR TITLE
add "whatlies" to spaCy universe

### PR DESCRIPTION
**Pull request by koaning**
We're releasing it on our side officially on the 16th of April. If possible, let's announce around the same time :)

## Description
This adds "whatlies" to the spaCy universe json.

### Types of change
This adds "whatlies" to the spaCy universe json.

## Checklist
- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/meta/universe.json [hunk 1]
```diff
@@ -1,5 +1,33 @@
 {
     "resources": [
+        {
+            "id": "whatlies",
+            "title": "whatlies",
+            "slogan": "Make interactive visualisations to figure out 'what lies' in word embeddings.",
+            "description": "This small library offers tools to make visualisation easier of both word embeddings as well as operations on them. There a convenient API to perform linear algebra as well as support for popular transformations like PCA/UMAP/etc.",
```
Comment (svlandeg) [inline website/meta/universe.json:7]: A verb is missing: "There a". Perhaps it also makes sense to include in the description how it uses spaCy? Is it mainly to use the prebuilt models ?

Comment (koaning) [reply website/meta/universe.json]: Ah yes. Dyslexia strikes again. Should be fixed now.

**Comment by koaning** [comment]
Also added a comment on spaCy support.

**Comment by svlandeg** [comment]
> We're releasing it on our side officially on the 16th of April. If possible, let's announce around the same time :)

Do you want us to hold off on the merge until then ?

**Comment by koaning** [comment]
Naa. Merging is fine. It's more that next week we're gonna have a youtube video + tweets + blogs on the thing. If you folks want to tweet about it, preferably wait 'till next monday.

**Comment by koaning** [comment]
@svlandeg just to check. i was hoping to announce this next week ... (13th of april actually) ... and I would love to also link to the spaCy universe ... I didn't see the website update, do your docs only build on a new relase?

**Comment by svlandeg** [comment]
It required some additional steps (and a mistake in the JSON we overlooked), but it should be live now!

**Comment by koaning** [comment]
Awesome, thanks! 

It does seem that the github shields can be buggy. Figured it was worth to mention: 

image

**Comment by ines** [comment]
@koaning Are you referring to the "no releases"? That's currently based on the GitHub releases and since those are [empty](https://github.com/rasahq/whatlies/releases), it shows "no releases". Maybe we should make it PyPi releases by default... but might be a bit weird because not everything is on PyPi and the buttons are otherwise all GitHub-related.

**Comment by koaning** [comment]
Ahhh! That explains it. I'd keep it attached as is. There's plenty of js/R tools also in the universe.

**Pull request merged**